src/StyleSheet.js

```diff
@@ -30,22 +29,26 @@ export default class StyleSheet {
     this.rules = Object.create(null)
     // Used to ensure correct rules order.
     this.rulesIndex = []
-    this.classes = Object.create(null)
+    this.classes = this.options.classes = Object.create(null)
```

Comment:
Oh, can't do the last one, classes is a hash of all classes for the sheet, it needs to be accessed from all RulesContainer instances. By passing a classes object I basically pass a registry for class names to all RulesContainer. Knowing that it is an internal implementation detail, is it actually that bad?